ingestr/src/adjust/helpers.py

```diff
@@ -0,0 +1,60 @@
+import pendulum
+import requests
+from dlt.sources.helpers.requests import Client
+
+
+class AdjustAPI:
+    def __init__(self, start_date, end_date, api_key):
+        self.start_date = start_date
+        self.end_date = end_date
+        self.api_key = api_key
+        self.default_dimensions = "campaign,day,app,store_type,channel,country"
+        self.default_metrics = (
+            "network_cost,all_revenue_total_d0,ad_revenue_total_d0,revenue_total_d0,"
+            "all_revenue_total_d1,ad_revenue_total_d1,revenue_total_d1,all_revenue_total_d3,"
+            "ad_revenue_total_d3,revenue_total_d3,all_revenue_total_d7,ad_revenue_total_d7,"
+            "revenue_total_d7,all_revenue_total_d14,ad_revenue_total_d14,revenue_total_d14,"
+            "all_revenue_total_d21"
+        )
```

Comment:
do not make them class attributes, instead make them variables outside of the class, and then accept metrics optionally from different functions.

e.g. the following code should work:
```python
client = AdjustAPI(api_key="my-api-key")
client.fetch_report_data(dimensions=['campaign', 'day'], metrics=['network_cost', 'revenue_total_d1'])
```

in order to get this to work, you need to change the functions.